feeds/ipq807x_v5.4/hostapd/files/hostapd.sh

```diff
@@ -74,7 +74,7 @@ hostapd_append_wpa_key_mgmt() {
 			append wpa_key_mgmt "OWE"
 		;;
 		psk2-radius)
-			append wpa_key_mgmt "WPA-PSK-SHA256"
+			append wpa_key_mgmt "WPA-PSK"
```

Comment:
I prefer to use WPA-PSK WPA-PSK-SHA256 instead.

Reply:
Finally I got access to push changes, similar PR:
https://github.com/Telecominfraproject/wlan-ap/pull/707

Reply:
I tried that but it doesn't work. I tested "wpa_key_mgmt=WPA-PSK WPA-PSK-SHA256". It failed with my iPhone15 Pro and my OnePlus12 phone. The phones will always choose 256

Reply:
Thanks, seems that your patch is better.